docs/topics/http/urls.txt

```diff
@@ -368,6 +368,24 @@ the following example is valid::
 In the above example, the captured ``"username"`` variable is passed to the
 included URLconf, as expected.
 
+Nested captured parameters are discarded by Django when urls are reversed,
+using only the outer parameters to find a reverse for a pattern. If you want
+to reverse such patterns, only pass the outer argument or named argument.
+
+For cases such as making a part of the pattern optional, use non-capturing
+arguments -- for example when a non-query page-specifier is optional::
+
+    # In urls.py
+    from django.conf.urls import url
+
+    urlpatterns = [
+        url(
```

Comment:
would be nice if this fit on 1 line like the other examples - we could drop name=... if it helps.